jechova.py

```diff
@@ -13,9 +14,13 @@
 
 
 if __name__ == '__main__':
-    force = '-f' in sys.argv
-    channel = sys.argv[1].lstrip('#')
-    ics_url = sys.argv[2]
+    p = ArgumentParser()
```

Comment:
Můžeme to pojmenovat "parser" místo "p"?

Reply:
ook